pkgs/development/libraries/protobufc/default.nix

```diff
@@ -19,6 +19,10 @@ stdenv.mkDerivation rec {
     hash = "sha256-TJCLzxozuZ8ynrBQ2lKyk03N+QA/lbOwywUjDUdTlbM=";
   };
 
+  postPatch = ''
+    sed -i 's,-lprotoc,-lprotoc -lprotobuf,' Makefile.am
```

Comment:
Couldn't you copy the full line to fix order ?

And you should first open PR and if merged use `fetchpatch` to take it here. Unless it does fix a immediate issue you didn't mentioned ?

Reply:
Full line is:
```
→       -lprotoc
```
(arrow denotes tab symbol). I think I would make this patch even more brittle if I include `\t' into sed command.

Reply:
For some reason, `msmtp` build-depends on `protobufc`, so to fix _and_ merge fix for `pkgsStatic.msmtp` I have to fix _and_ merge `protobufc` which I have no use. 

I am okay with making patch for upstream and `fetchpatch`'ing it, though. I'd rather not wait when and if upstream merges it, though.

Reply:
ok just fetch this patch https://github.com/protobuf-c/protobuf-c/commit/a6c9ea5207aeac61c57b446ddf5a6b68308881d8.patch